tests/unit/test_backends.py

```diff
@@ -190,47 +194,8 @@ class TestFileSystemBackend(TestAbstractBackend):
     """
     def setUp(self):
         self._dataDir = os.path.join("tests", "data")
-        self._referencesDir = os.path.join(self._dataDir, "references")
```

Comment:
We're not using any of these --- it's confusing to have a bunch of code lying around that's not being used for anything.